tests/test_simpleButler.py

```diff
@@ -166,10 +166,10 @@ def testDatasetTransfers(self):
     def testComponentExport(self):
         """Test exporting component datasets and then importing them.
 
-        This test intentionally does not depend on whether just the component
-        is exported and then imported vs. the full composite dataset, because
-        I don't want it to assume more than it needs to about the
-        implementation.
+        This test exports component datasets and then checks via the parent
+        dataset type, since this is the only valid way to handle component
+        exports (since Registry cannot record just a component without a
+        parent, even if Datastore can).
```

Comment:
I'm effectively disabling the ability of datastore to store a bare component in #737.

Reply:
Where does that leave this test?  It seems to still be working.

Reply:
I removed the test in datastore, although I didn't make datastore.put reject a component dataset type. Maybe that was a mistake, I hadn't realized we had other tests that did disassembly outside datastore.

Reply:
I'm a bit confused what is happening in this test since registry doesn't do composite disassembly.

Reply:
Ah, it looks like the doctring had just gotten out of date, and I took it at face-value when I updated it.  It just exports the composite and checks that the components are accessible after import.  But this ticket makes that check for components impossible to do directly, and doing it indirectly via the parent datasets just makes this test the same as some others, so I'll delete it.